docs/how-we-work.md

```diff
@@ -0,0 +1,131 @@
+
+If you believe an issue _should_ be addressed before merge, mark that comment with a :rotating_light:.
+
+When finalizing your review:
+
+* "Approve" means "click merge whenever you're ready, I'm okay with this shipping exactly as I see it here if you are."
+* "Comment" means "I've noted things that you may want to consider doing differently. Feel free to merge if you disagree, but further conversation might be useful to bring me on board."
+* "Request changes" means "I believe that merging this right now would break something. Dismiss my review once you've addressed the urgent issues that I've identified."
+
+## How we ship
+
+The github package ships as a bundled part of Atom, which affects the way that our progress is delivered to users. After using `apm` to publish a new version, we also need to add a commit to [Atom's `package.json` file](https://github.com/atom/atom/blob/master/package.json#L114) to make our work available.
+
+When the team is preparing to ship a new version of Atom, run `apm publish minor` and update `package.json` on Atom's master branch to reference the new version. This will ship our work to Atom's [beta channel](https://atom.io/beta) and allow a smaller subset of our users to discover regressions before we release it to the full Atom user population.
+
+When you've merged substantial new functionality, consider running `apm publish minor` and updating `package.json` on Atom's master branch outside of the Atom release cycle, to give the rest of the Atom team time to dogfood the change internally and weigh in with opinions.
+
+After shipping a minor version release for either of the above situations, create and push a release branch from that version's tag:
+
+```sh
+$ apm publish minor
+version 0.11.0
+$ git branch 0.11-releases && git push -u origin 0.11-releases
+```
+
+When you merge a fix for a bug, cherry-pick the merge commit onto to the most recent release branch, then run `apm publish patch` and update `package.json` on the most recent beta release branch on the `atom/atom` repository. This will ensure bug fixes are delivered to users on Atom's stable channel as part of the next release.
+
+When you merge a fix for a **security problem**, a **data loss bug**, or fix a **crash** or a **lock-up** that affect a large portion of the user population, run `apm publish patch` and update `package.json` on the most recent beta _and_ stable release branches on the `atom/atom` repository. Consider advocating for a hotfix release of Atom to deliver these fixes to the user population as soon as possible.
```

Comment:
Should this be hotfixed to the branch of the currently released `minor` version on Atom stable? As written this could involve an update of several minor versions of `github` to the `stable` branch for a "hotfix".

Reply:
Ah, good catch. I think I confused myself quite a bit when I was transferring this from our notes 😄 

Also: this is kind of cumbersome. I actually advocated for us _not_ to do this way back when we were getting started because it adds a lot of complexity and ceremony. If we end up having to do this more than once or twice I'll likely get us a script we can use to make sure the fixes go to the right branches and such. 🤔